src/views/Home.vue

```diff
@@ -9,10 +9,13 @@
                 <span class="has-text-primary">Group</span> links,
                 <span class="has-text-primary">Save</span> &
                 <span class="has-text-primary">Share</span> them with the
```

Comment:
Can we add a break after the main title. Just a little more space would be nice.